index.js

```diff
@@ -96,11 +107,61 @@ async function terraform(args) {
   if (tfi.status > 0) {
     tf_init = status_failed;
     core.setFailed(`Failed to initialize terraform [err:${tfi.status}]`);
-    terraformDoApply = 'false';
+    terraformDoApply = false;
   } else {
     tf_init = status_success;
   }
 
+  /* VARIABLES START */
+  core.startGroup('Assign terraform variables');
+  if (terraformVariables) {
+    let variables = JSON.parse(terraformVariables);
+    for (let key in variables) {
+      if (Object.prototype.hasOwnProperty.call(variables, key)) {
+        shell.setVariable(`TF_VAR_${key}`, variables[key]);
+        core.info(`Assigned variable ${key} with value ${variables[key]}`);
```

Comment:
Should this be info or debug level?
```suggestion
        core.debug(`Assigned variable ${key} with value ${variables[key]}`);
```
ℹ️ https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging

Reply:
I'd rather always see whats passed as vars instead of having to enable debugging separately, but fine either way